### PR TITLE
Fix netcoreapp3.0

### PR DESCRIPTION
**Pull request by ryanbrandenburg**
Fix some of the tests that started to fail after the move to netcoreapp3.0.

### CODE REVIEW

**Comment by ryanbrandenburg** [comment]
The VSTS failure is a known issue.

File: scripts/Run-Angular-Locally.ps1 [hunk 1]
```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env powershell
```
Comment (ryanbrandenburg) [inline scripts/Run-Angular-Locally.ps1]: I think I can combine Angular, React and ReactRedux into `Run-Spa-Locally.ps1` and just pass it the name of the SPA.

Comment (natemcmaster) [reply scripts/Run-Angular-Locally.ps1]: By the way, change the shebang to `#!/usr/bin/env pwsh -c`. They renamed powershell => pwsh.

File: src/Microsoft.DotNet.Web.ProjectTemplates/Microsoft.DotNet.Web.ProjectTemplates.csproj [hunk 2]
```diff
@@ -13,6 +13,7 @@
   <!-- Set this last to ensure the properties get the final versions which may be overridden by CI. -->
   <PropertyGroup>
     <GeneratedContentProperties>
+      MicrosoftAspNetCoreAllPackageVersion=$(MicrosoftAspNetCoreAllPackageVersion);
```
Comment (natemcmaster) [inline src/Microsoft.DotNet.Web.ProjectTemplates/Microsoft.DotNet.Web.ProjectTemplates.csproj]: Can you remove this? I intentionally dropped references to Microsoft.AspNetCore.All in the templating repo.

File: src/Microsoft.DotNet.Web.Spa.ProjectTemplates/Angular-CSharp.csproj.in [hunk 3]
```diff
@@ -13,8 +13,13 @@
     <BuildServerSideRenderer>false</BuildServerSideRenderer>
     <!-- aspnet/BuildTools#662 Don't police what version of NetCoreApp we use -->
     <NETCoreAppMaximumVersion>99.9</NETCoreAppMaximumVersion>
+    <RuntimeFrameworkVersion>${MicrosoftNETCoreApp30PackageVersion}</RuntimeFrameworkVersion>
   </PropertyGroup>
 
+  <ItemGroup>
+    <DotNetCoreRuntime Include="${MicrosoftNETCoreApp30PackageVersion}" />
```
Comment (natemcmaster) [inline src/Microsoft.DotNet.Web.Spa.ProjectTemplates/Angular-CSharp.csproj.in]: Remove this. This is a KoreBuild-ism, not a built-in SDK feature. This shouldn't be in the templates we give to users.

File: src/Microsoft.DotNet.Web.Spa.ProjectTemplates/Angular-CSharp.csproj.in [hunk 4]
```diff
@@ -13,8 +13,13 @@
     <BuildServerSideRenderer>false</BuildServerSideRenderer>
     <!-- aspnet/BuildTools#662 Don't police what version of NetCoreApp we use -->
     <NETCoreAppMaximumVersion>99.9</NETCoreAppMaximumVersion>
```
Comment (natemcmaster) [inline src/Microsoft.DotNet.Web.Spa.ProjectTemplates/Angular-CSharp.csproj.in]: We should probably remove these 3 lines from templates. They are workarounds for tests, but they are not content we actually want in the .csproj files we generate for users.

File: test/Templates.Test/Templates.Test.csproj [hunk 5]
```diff
@@ -4,7 +4,7 @@
     <DefineConstants>$(DefineConstants);XPLAT</DefineConstants>
     
     <!-- Workaround until https://github.com/aspnet/Common/pull/385 is done -->
-    <DefineConstants>$(DefineConstants);NETCOREAPP2_2</DefineConstants>
+    <DefineConstants>$(DefineConstants);NETCOREAPP3_0</DefineConstants>
```
Comment (natemcmaster) [inline test/Templates.Test/Templates.Test.csproj]: You can remove this altogether now that https://github.com/aspnet/Common/pull/385 is fixed.

**Comment by ryanbrandenburg** [comment]
Superseded by #647.

**Pull request closed without merging**